### PR TITLE
A few small copy edits for the website home page

### PR DESCRIPTION
**Pull request by ryankeairns**
:shipit: 

Signed-off-by: Ryan Keairns <rkeairns@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/index.html.slim [hunk 1]
```diff
@@ -158,7 +158,7 @@ section.main-section.first
           h4 Isolated Build Environment
           p
             | Define your entire app in a plan file and generate reproducible,
-              trustworthy, versioned builds in a clean-room environment.
+              versioned builds in a clean-room environment.
```
Comment (ryankeairns) [inline www/source/index.html.slim]: Not that they aren't trustworthy 😉 , but this was a bit wordy and it was causing the line to wrap and the remaining bullet features were misaligned.

**Comment by reset** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by adamhjk** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/162027586).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/162027586).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265
